src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -57,7 +58,7 @@
 
 logger = logging.getLogger(__name__)
 
-MONITOR_CODEC = get_codec("ingest-monitors")
+MONITOR_CODEC: Codec[IngestMonitorMessage] = get_codec("ingest-monitors")
```

Comment:
Helps the codec understand it's going to be encoding / decoding a `IngestMonitorMessage`